$MirOS: ports/emulators/qemu/patches/patch-target-ppc_op_helper_c,v 1.2 2005/05/03 19:30:05 tg Exp $
--- target-ppc/op_helper.c.orig	Sun Sep  4 17:11:09 2005
+++ target-ppc/op_helper.c	Tue Nov 15 11:39:51 2005
@@ -18,6 +18,7 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  */
 #include <math.h>
+#include <float.h>
 #include "exec.h"
 
 #define MEMSUFFIX _raw
